src/contracts/index.html

```diff
@@ -1,6 +1,14 @@
 ---
 layout: skinny
+
 title: California’s Contracts—for U.S. transit agencies
+description: >
+  California has Master Service Agreements (MSA) available to help you modernize
+  your transit system, most of which are available to public transportation
+  providers. The Mobility Marketplace serves as Cal-ITP’s online catalog where
+  transit providers can browse available goods and services to build a system
+  that suits their needs best.
+
```

Comment:
<img width="747" alt="image" src="https://github.com/user-attachments/assets/4b1e8d60-5e19-4977-81a6-b16256390e51">
@mrose914 FYI - this is how these descriptions show up